docs/conf.py

```diff
@@ -43,7 +43,7 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = ["**test**"]
+exclude_patterns = []
```

Comment:
I had originally excluded tests b/c I thought that doing so would stop sphinx from compiling documentation for the tests.

Am I not understanding correctly? What does removing this do?

Reply:
This is correct in that it avoids rendering any of the `pycytominer.tests.*.rst` documents as HTML. I changed this to include `pycytominer.tests.test_data.rst` when documentation is rendered, providing context and information on the sample data. We could store this elsewhere too, if we shouldn't render those docs. For a visual of what renders to HTML when removing the `exclude_pattern`, see the following screenshot: 

![image](https://user-images.githubusercontent.com/3738008/188234330-d3bf22c0-3134-44c8-948c-afd5b947a94b.png)

Reply:
Gotcha! Yes, I think it's best to remove tests from the docs. I do not think I've ever seen tests in docs before.

Reply:
Understood - I'll add the exclude flag for that suite of documents back in with a related commit.